validation/taylor_green_vortex_2d/validation_taylor_green_vortex_2d.jl

```diff
@@ -2,7 +2,8 @@ using TrixiParticles
 
 # ==========================================================================================
 # ==== Resolution
-particle_spacings = [0.02, 0.01, 0.005]
+# particle_spacings = [0.02, 0.01, 0.005]
+particle_spacings = [0.02]
```

Comment:
```suggestion
particle_spacing = 0.02
```
According to the discussion in another PR.